action-scripts/brownie/scripts/report_gauges.py

```diff
@@ -146,15 +156,17 @@ def _parse_removed_transaction(transaction: dict, **kwargs) -> Optional[dict]:
     """
     Parse a gauge remover transaction and return a dict with parsed data.
     """
-    if network.is_connected():
-        network.disconnect()
-    network.connect(CHAIN_MAINNET)
     input_values = transaction.get("contractInputsValues")
     if not input_values or not isinstance(input_values, dict):
         return
     encoded_data = input_values.get("data")
     if not encoded_data:
         return
+
+    if network.is_connected():
```

Comment:
Why moved here?

Reply:
Because the scripts ran really slow as brownie was always disconnecting/reconnecting even on reports with nothing to do.  This made things 2x faster.

Reply:
Good find then!